### ISSUE TITLE
Sort out custom css

### ISSUE THREAD

**Issue by ir-g** [Kefwmag/kefwmag.github.io#70]
We are fetching multiple CSS files. For smaller css files we should consider directly adding them via includes into the head of the documents.

**Comment by ChilliByte** [comment]
Yes I was considering using YAML front matter to generate custom CSS

**Comment by ir-g** [comment]
@ChilliByte Yeah - Im hoping to make authors, editions and categories have their own custom css declared in some their own file, as well as generating shortcodes for authors, and upgrading the formatter to allow us to do bulk page generation, and allowing automatic adding to github.

**Comment by ChilliByte** [comment]
I was with you until "bulk page generation, and allowing automatic adding to github."

**Comment by ir-g** [comment]
@ChilliByte Erm. My english went downhill from there to be honest - You can ignore that bit.

Eventually I'll add a feature to the formatter where you can login and upload posts _from_ the formatter.

**Comment by ChilliByte** [comment]
WHOA WHAT
ON A STATIC SITE
HOW

**Comment by ir-g** [comment]
Github API magic ;)

**Comment by ChilliByte** [comment]
@isaacrg how does the custom css work now

**Comment by ir-g** [comment]
@chillibyte For now, it works like usual. I'm not sure of a way of approaching custom CSS properly.

**Comment by ChilliByte** [comment]
the old stylesheets dont work with your new system - `header` and `section` dont have classes

**Comment by ChilliByte** [comment]
see #92 for custom css

I know its exam season, but how is the front matter generator coming along?

**Comment by isaacrg** [comment]
I've done little on it, in part because I suspect we may need to create an application that handles editing of posts, as well.

**Comment by ir-g** [comment]
I've done little on it, in part because I suspect we may need to create an application that handles editing of posts, as well - for when the website is handed over to the next group of writers/etc
